osquery/tables/system/windows/kva_speculative_info.cpp

```diff
@@ -0,0 +1,150 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+/**
+ * Copyright 2018 Alex Ionescu. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
```

Comment:
This indemnification clause / below license should be compatible with ASL2.0. You should place the above copyright above the project copyright at the top.